geode-benchmarks/build.gradle

```diff
@@ -23,16 +23,29 @@ version '1.0-SNAPSHOT'
 sourceCompatibility = 1.8
 
 def outputDir = project.hasProperty('outputDir') ? project.findProperty('outputDir') : new File(project.buildDir, "benchmarks_" + getDate()).getAbsolutePath()
-def geodeVersion = project.hasProperty('geodeVersion') ? project.findProperty('geodeVersion') : '1.+'
+def geodeVersion = project.hasProperty('geodeVersion') ? project.findProperty('geodeVersion') : '1.13.+'
 
 def getDate() {
   new Date().format('yyyyMMddHHmmss')
 }
 
 
 repositories {
+  // in CI we want to search mavenLocal for geode
   mavenLocal()
-  mavenCentral()
+  // non-geode stuff can come from mavenCentral
+  mavenCentral() {
+    content {
+      excludeGroup 'org.apache.geode'
+    }
+  }
+  // when developing this project, use Geode develop branch snapshot
+  maven {
+    url "https://maven.apachegeode-ci.info/snapshots"
+    content {
+      includeGroup 'org.apache.geode'
+    }
+  }
```

Comment:
now that we are looking for an unreleased version of Geode (1.13+) we have to look in our in-house Maven repo

Reply:
This is resolved by mavenLocal. The scripts will build any version of geode and put it in the local repo. Again revert this change.

Reply:
That works in CI but it doesn't, in general, work in the dev environment.

Are you suggesting a workflow in the dev environment, where you clone the `geode` repo and then `./gradlew publishToMavenLocal` before working in the `geode-benchmarks` repo?

Reply:
The latest commit eliminates the Apache snapshots repo from the search

Reply:
Whelp now CI can't find an appropriate Geode version https://travis-ci.org/github/apache/geode-benchmarks/builds/695464444

<img width="914" alt="image" src="https://user-images.githubusercontent.com/4002/83951336-a113cc00-a7e5-11ea-94c4-41980b054b8b.png">

Reply:
Before this PR, it just so happened that `geode-benchmarks` did not depend on an unreleased Geode version. So it was always able to find a sufficient Geode version on Maven Central.

With this PR that's no longer the case, since we now depend on Geode 1.13.0 or later and no 1.13.x Geode version exists in Maven Central.

So I am reinstating the search of the Apache Snapshots repo until I can think of a better solution for CI.

Reply:
You don't need snapshot repos. You can just specify a branch and the benchmarks scripts will build that branch. Versions should only be used for released versions.

Reply:
That doesn't work in CI (see above)

Reply:
We were talking about two different CI. The Geode CI that runs the benchmarks relies on the original behavior. Let's fix the Travis CI you are running into.